packages/block-editor/src/hooks/block-bindings.js

```diff
@@ -128,38 +189,46 @@ function BlockBindingsAttribute( { attribute, binding, fieldsList } ) {
 				>
 					{ isSourceInvalid
 						? __( 'Invalid source' )
-						: fieldsList?.[ sourceName ]?.[ args?.key ]?.label ||
-						  sourceProps?.label ||
+						: source?.data?.find(
+								( item ) => item.key === args?.key
```

Comment:
needs `isSelected` call
```suggestion
								( item ) =>
									source.isSelected( { item, binding } ) ??
									item.key === args?.key
```

Reply:
On second thought, why does this display a field label at all? Shouldn't this always display the binding source label? A field label is out of context here.

Reply:
We moved to a non using `isSelected`. The field has been displayed since last versions. 😅